db/migrate/20210908075830_create_profiles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProfiles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :profiles do |t|
+      t.integer   :sex,         null: false
+      t.datetime  :birthday,    null: false
```

Comment:
[imo]
誕生日は時刻が必要ないので、`datetime`型よりも`date`型の方が適切だと思いました
```suggestion
      t.date  :birthday,    null: false
```